src/icalendar/prop.py

```diff
@@ -258,12 +267,16 @@ def to_ical(self):
         dts_ical = (dt.to_ical() for dt in self.dts)
         return b",".join(dts_ical)
 
-    @staticmethod
-    def from_ical(ical, timezone=None):
+    @classmethod
+    def from_ical(cls, ical, unit_type=None, timezone=None):
```

Comment:
I think we should keep the old order of arguments, this breaks backwards compatibility otherwise.